source/_integrations/modbus.markdown

```diff
@@ -1001,6 +1001,10 @@ sensors:
       description: The maximum allowed value of a sensor. If value > max_value --> max_value. Can be float or integer
       required: false
       type: float
+    nan_value:
+      description: The NaN value if the sensor has an explicit NaN return. Can be float, integer or in hex notation
+      required: false
+      type: int
```

Comment:
It does not tell anything about the purpose of the parameter and what it does.

Reply:
Better like this?
If a modbus sensor has an explicit NaN value, this optional parameter gives the ability to specify it, such that the modbus sensor returns "unavailable" when NaN is returned